projects/angular/src/layout/main-container/_properties.header.scss

```diff
@@ -4,59 +4,55 @@
+      --clr-header-3-bg-color: #{tokens.$cds-global-color-azure-900};
+      --clr-header-4-bg-color: var(--clr-header-3-bg-color);
+      --clr-header-5-bg-color: var(--clr-header-3-bg-color);
+      --clr-header-6-bg-color: var(--clr-header-bg-color);
+      --clr-header-7-bg-color: var(--clr-header-2-bg-color);
+      --clr-header-font-color: #{tokens.$cds-global-color-construction-100};
+      --clr-header-font-color-hover: #{tokens.$cds-global-color-construction-50};
+      --clr-header-title-font-weight: #{tokens.$cds-global-typography-subsection-font-weight};
+      --clr-header-title-font-family: #{tokens.$cds-global-typography-header-font-family};
+      --clr-header-title-font-size: #{tokens.$cds-global-typography-subsection-font-size};
+      --clr-header-title-line-height: #{tokens.$cds-global-typography-subsection-line-height};
+      --clr-header-title-letter-spacing: #{tokens.$cds-global-typography-subsection-letter-spacing};
+      --clr-header-branding-min-width: #{mixins.baselinePx(204)};
+      --clr-header-branding-padding: 0 #{tokens.$cds-global-space-9};
+      --clr-header-branding-icon-gap: #{tokens.$cds-global-space-5};
+      --clr-header-branding-logo-size: #{tokens.$cds-global-space-11};
+      --clr-header-nav-link-padding: #{tokens.$cds-global-space-8} #{tokens.$cds-global-space-7};
+      --clr-header-nav-link-line-height: #{tokens.$cds-global-space-9};
+      --clr-header-nav-link-font-size: #{tokens.$cds-global-typography-font-size-4};
+      --clr-header-nav-text-font-weight: #{tokens.$cds-global-typography-font-weight-regular};
+      --clr-header-nav-text-font-weight-active: #{tokens.$cds-global-typography-font-weight-semibold};
+      --clr-header-search-height: #{tokens.$cds-global-space-11};
+      --clr-header-search-icon-size: #{tokens.$cds-global-space-9};
+      --clr-header-search-margin-left: #{tokens.$cds-global-space-5};
+      --clr-header-search-margin-right: #{tokens.$cds-global-space-7};
+      --clr-header-search-gap: #{tokens.$cds-global-space-5};
+      --clr-header-search-input-font-size: #{tokens.$cds-global-typography-font-size-4};
+      --clr-header-search-input-font-color: #{tokens.$cds-global-color-white};
+      --clr-header-search-input-font-weight: #{tokens.$cds-global-typography-font-weight-semibold};
+      --clr-header-search-placeholder-font-weight: #{tokens.$cds-global-typography-font-weight-regular};
+      --clr-header-search-placeholder-text-color: #{tokens.$cds-global-color-construction-400};
+      --clr-header-search-border-active: #{tokens.$cds-global-space-1} solid Highlight;
+      --clr-header-search-border-hover: #{tokens.$cds-global-space-1} solid #{tokens.$cds-global-color-construction-400};
+      --clr-header-action-caret-right: #{tokens.$cds-global-space-7};
+      --clr-header-action-caret-size: #{tokens.$cds-global-space-7};
+      --clr-header-action-icon-size: #{tokens.$cds-global-space-9};
+      --clr-header-action-caret-gap: #{tokens.$cds-global-space-5};
```

Comment:
This has been designed to expose a property, as defined in spec.
I view it as improvement over the old API. And it will directly be used during the layout refactoring.